libs/platform/src/lib/table/components/table-row/table-row.component.html

```diff
@@ -163,7 +163,13 @@
         <ng-template #primaryTextContainer>
             <div
                 [class.fd-table__text]="column.applyText"
-                [class.fd-table__text--no-wrap]="column.noWrap"
+                [class.fd-table__text--no-wrap]="column.applyText"
+                [class.fd-table__cell--truncate-txt]="column.applyText"
+                [attr.title]="
+                    tableTextContainer.offsetWidth < tableTextContainer.scrollWidth
```

Comment:
Can you also optimize this? `offsetWidth`, `innerText` and `scrollWidth` can potentially cause forced reflow especially this is changed frequently.

Reply:
we'll address this in a later PR